packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -392,6 +405,25 @@ function FormatButton({icon, format}: SlateBlockButtonProps) {
   )
 }
 
+interface SlateBlockButtonWithChildrenProps extends BaseToolbarButtonProps {
+  readonly format: Format
+}
+
+function FormatButtonWithChildren({format, children}: SlateBlockButtonWithChildrenProps) {
+  const editor = useSlate()
+
+  return (
+    <ToolbarButtonWithChildren
+      active={isFormatActive(editor, format)}
+      onMouseDown={e => {
```

Comment:
Is this a conscious decision to use `onMouseDown` istead of `onMouseClick`?

Reply:
They seemed to use this terminology especially with slate.js. So, did not invent it but adapted to it.